src/orion/benchmark/task/base.py

```diff
@@ -23,8 +23,8 @@ class BaseTask(ABC):
        implementation can have its own parameters.
     """
 
-    def __init__(self, max_trials, **kwargs):
-        self.trials_num = max_trials
+    def __init__(self, max_trials: int, **kwargs):
```

Comment:
Please avoid typing unless the whole file is using it, to maintain coherence in the module's code style.

Reply:
Adding types everywhere in that module as a solution :)